ikarus/finiteelements/mechanics/linearelastic.hh

```diff
@@ -26,18 +26,21 @@
 
 namespace Ikarus {
 
-template <typename PreFE, typename FE>
+template <typename PreFE, typename FE, typename PRE>
 class LinearElastic;
 
 /**
  * \brief A PreFE struct for linear elastic elements.
  */
+template <typename MAT>
+requires(MAT::strainTag == StrainTags::linear)
```

Comment:
Can you create a concept from this? `concept GeometricallyLinearMaterial` or similar?